PlexRequests.UI/Modules/RequestsModule.cs

```diff
@@ -84,7 +84,7 @@ private Response GetMovies()
             var dbMovies = Service.GetAll().Where(x => x.Type == RequestType.Movie);
             if (settings.UsersCanViewOnlyOwnRequests && !isAdmin)
             {
-                dbMovies = dbMovies.Where(x => x.RequestedBy.Equals(Session[SessionKeys.UsernameKey].ToString(), StringComparison.OrdinalIgnoreCase));
+                dbMovies = dbMovies.Where(x => x.UserHasRequested(Session[SessionKeys.UsernameKey].ToString()));

```

Comment:
Since we are using `Session[SessionKeys.UsernameKey]` a few times in this method, we should store it in it's own local variable so we don't have to keep querying the session.